src/write.c

```diff
@@ -749,32 +849,52 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
     }
 
     const avifImage * firstCell = cellImages[0];
+    const avifImage * topRightCell = cellImages[gridCols - 1];
+    const avifImage * bottomLeftCell = cellImages[cellCount - gridCols];
```

Comment:
Optional: I just realized that we can replace `topRightCell` and `bottomLeftCell` with `bottomRightCell`:

```
    const avifImage * bottomRightCell = cellImages[cellCount - 1];
```

The top-right cell is the first cell of the right-most column, and the bottom-left cell is the first cell of the bottom-most row. That's why we think of them first. But if the cells in the image grid satisfy the width and height requirements, we can also use the bottom-right cell because it's in both the right-most column and the bottom-left cell.